core/server/middleware/middleware.js

```diff
@@ -111,7 +111,7 @@ middleware = {
         });
 
         if (subPath.indexOf('/ghost/api/') === 0
-            && path.indexOf('/ghost/api/v0.1/authentication/') !== 0) {
+            && (path.indexOf('/ghost/api/v0.1/authentication/') !== 0 || req.method === 'PUT')) {
```

Comment:
@ErisDS I think this is what you were going for? I'm not 100% sure how to test that it works though.....